examples/docker-compose.yaml

```
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  gqlgen-server:
+    build:
+      context: gqlgen-service
+    expose:
+      - 8080
+  gophers-server:
+    build:
+      context: graph-gophers-service
+    expose:
+      - 8080
+  nodejs-server:
+    build:
+      context: nodejs-service
+    expose:
+      - 8080
+  gateway:
+    image: ghcr.io/movio/bramble
+    volumes:
+      - ${PWD}/example-config.json:/config.json
```

Where does the `PWD` value come from? Are you using a `.env` file that's not commited?
```suggestion
      - ./example-config.json:/config.json
```